synfig-studio/src/synfigapp/action_system.h

```diff
@@ -105,6 +106,8 @@ class System : public etl::shared_object, public sigc::trackable
 	*/
 
 public:
+		static bool repeated_action;
```

Comment:
It's better to move this to private non-static variables.
Each file has its own ActionSystem. If we try to work with them in parallel (from different threads for example), in the case of static members, this will lead to undefined behavior.

Reply:
yes, I was initially thinking that because it probably isn't possible to have more than one of these "repeated actions" then a static variable would be enough to be used by each "repeated action". But yes I can see how this could get messy and possibly lead to some undefined behavior.

switched over to _public_ non-static variables. As they need to be set and gotten from outside, however if its better to just keep them private and write setters and getters please let me know :)